doc/source/whatsnew/v0.18.0.txt

```diff
@@ -184,6 +184,22 @@ In addition, ``.round()``, ``.floor()`` and ``.ceil()`` will be available thru t
    s
    s.dt.round('D')
 
+.. _whatsnew_0180.enhancements.xarray:
+
+to_xarray
+^^^^^^^^^
+
+In a future version pandas we will be deprecating ``Panel`` and other > 2 ndim objects. In order to provide for continuity,
```

Comment:
"In a future version pandas we will " should be either "In a future version of pandas, we will" or "in a future version, pandas will"